.editorconfig

```diff
@@ -0,0 +1,74 @@
+# Visual Studio generated .editorconfig file with C++ settings.
+root = true
+
+[*.{c++,cc,cpp,cppm,cxx,h,h++,hh,hpp,hxx,inl,ipp,ixx,tlh,tli}]
+
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+charset = utf-8
+
+# Visual C++ Code Style settings
+
+cpp_generate_documentation_comments = doxygen_triple_slash
+
+# Visual C++ Formatting settings
+
+cpp_indent_braces = false
+cpp_indent_multi_line_relative_to = statement_begin
```

Comment:
Did you change this from the default one? If so, why? When I auto-generated the file on my machine it is put to:
`cpp_indent_multi_line_relative_to = innermost_parenthesis`

Reply:
same story here - yes, I changed this on purpose, because it better fits to "my preferences" and the existing code-style.